Public/Src/Cache/ContentStore/Distributed/Redis/RedisContentLocationStoreConfiguration.cs

```diff
@@ -91,6 +91,11 @@ public sealed class RedisContentLocationStoreConfiguration : LocalLocationStoreC
         /// </summary>
         public ContentLocationMode ReadMode { get; set; } = ContentLocationMode.Redis;
 
+        /// <summary>
+        /// Indicates the window size for executing eviction.
+        /// </summary>
+        public int EvictionWindowSize { get; set; } = 500;

```

Comment:
We need to change distributed settings to actually make this configurable from cloudbuild configuration.